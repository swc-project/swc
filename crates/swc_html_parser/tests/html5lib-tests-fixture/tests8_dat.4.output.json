{
  "type": "Document",
  "span": {
    "start": 1,
    "end": 34
  },
  "mode": "quirks",
  "children": [
    {
      "type": "Element",
      "span": {
        "start": 0,
        "end": 0
      },
      "tagName": "html",
      "namespace": "http://www.w3.org/1999/xhtml",
      "attributes": [],
      "children": [
        {
          "type": "Element",
          "span": {
            "start": 0,
            "end": 0
          },
          "tagName": "head",
          "namespace": "http://www.w3.org/1999/xhtml",
          "attributes": [],
          "children": [],
          "content": null,
          "isSelfClosing": false
        },
        {
          "type": "Element",
          "span": {
            "start": 0,
            "end": 0
          },
          "tagName": "body",
          "namespace": "http://www.w3.org/1999/xhtml",
          "attributes": [],
          "children": [
            {
              "type": "Element",
              "span": {
                "start": 8,
                "end": 34
              },
              "tagName": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "attributes": [],
              "children": [
                {
                  "type": "Text",
                  "span": {
                    "start": 13,
                    "end": 14
                  },
                  "data": "x",
                  "raw": "x"
                },
                {
                  "type": "Element",
                  "span": {
                    "start": 14,
                    "end": 25
                  },
                  "tagName": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "attributes": [],
                  "children": [],
                  "content": null,
                  "isSelfClosing": false
                },
                {
                  "type": "Text",
                  "span": {
                    "start": 25,
                    "end": 34
                  },
                  "data": "xx",
                  "raw": "xx"
                }
              ],
              "content": null,
              "isSelfClosing": false
            },
            {
              "type": "Element",
              "span": {
                "start": 1,
                "end": 8
              },
              "tagName": "table",
              "namespace": "http://www.w3.org/1999/xhtml",
              "attributes": [],
              "children": [],
              "content": null,
              "isSelfClosing": false
            }
          ],
          "content": null,
          "isSelfClosing": false
        }
      ],
      "content": null,
      "isSelfClosing": false
    }
  ]
}
