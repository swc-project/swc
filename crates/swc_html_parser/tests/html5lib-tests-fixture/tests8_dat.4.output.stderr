(1,7): expected-doctype-but-got-start-tag
(1,12): foster-parenting-start-tag
(1,13): foster-parenting-character
(1,18): foster-parenting-start-tag
(1,24): foster-parenting-end-tag
(1,25): foster-parenting-start-tag
(1,32): foster-parenting-end-tag
(1,32): unexpected-end-tag
(1,33): foster-parenting-character
(1,33): eof-in-table